plugins/kotlin-as-java/src/main/kotlin/KotlinAsJavaPageContentBuilder.kt

```diff
@@ -0,0 +1,185 @@
+package org.jetbrains.dokka.kotlinAsJava
+
+import org.jetbrains.dokka.links.DRI
+import org.jetbrains.dokka.model.Documentable
+import org.jetbrains.dokka.model.Function
+import org.jetbrains.dokka.model.TypeWrapper
+import org.jetbrains.dokka.model.doc.DocTag
+import org.jetbrains.dokka.pages.*
+import org.jetbrains.dokka.transformers.psi.JavaTypeWrapper
+import org.jetbrains.dokka.utilities.DokkaLogger
+
+class KotlinAsJavaPageContentBuilder(
```

Comment:
Instead of copying half of the `DefaultPageContentBuilder`, it should be opened and subtyped here instead IMHO. @Kordyjan WDYT?

Reply:
Fixed.

Reply:
Not in this version as I see it